indico/web/templates/forms/principal_list_widget.html

```diff
@@ -26,11 +26,15 @@
         setupPrincipalListWidget({
             fieldId: {{ field.id | tojson }},
             eventId: {{ (field.event.id if field.event else none) | tojson }},
-            openImmediately: {{ field.open_immediately | tojson }},
-            groups: {{ field.groups | tojson }},
-            allowExternal: {{ field.allow_external | tojson }},
-            allowNetworks: {{ field.allow_networks | tojson }},
-            networks: {{ field.ip_networks | sort(attribute='name') | tojson }}
+            openImmediately: {{ field.open_immediately | default(False) | tojson }},
+            withGroups: {{  (field.groups or field.allow_groups) | tojson }},
+            withExternalUsers: {{ (field.allow_external or field.allow_external_users) | tojson }},
```

Comment:
We should probably settle on a naming convention here. I was keeping `withX` on the templating and react side and `allowX` Python-wise. But it's starting to get confusing, can we perhaps keep one of them across all places?